apps/lukso-status/src/app/modules/launchpad/launchpad/services/keygen.service.ts

```diff
@@ -0,0 +1,20 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class KeygenService {
+  httpClient: HttpClient;
+  constructor(httpClient: HttpClient) {
+    this.httpClient = httpClient;
+  }
+
+  genereateKeys(password: string, network: string, amountOfValidators: string) {
+    return this.httpClient.post('/api/launchpad/generate-keys', {
```

Comment:
I think that entire setup could be handled via WS instead of REST, but to simplify V1 alpha release lets stay with REST-like.
BTW you can create const/enum for endpoints

Reply:
Yes I agree, the goal should be websockets. Once we have a working product, we should change this.